app/components/question_component/question_component.en.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  answers:
+    zero: "0 answers"
```

Comment:
Could put something like "No answers" or "Unanswered" here.

Reply:
Zarro answoors found

Reply:
This technically doesn't appear anyway (answer count must be positive to show uppp), it's just added to match all numerical translation values

Reply:
then i would suggest to change it to something more fun

```suggestion
    zero: "zefix, do is jo nix"
```